### ISSUE TITLE
Feature Request: Combine Tracks

### ISSUE THREAD

**Issue by julianstanley** [igvteam/igv.js#1467]
I use two wig files that represent 5'-end and 3'-end RNA sequencing counts. In regular IGV, I use "overlay tracks" to combine them, like this:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/22749289/155610555-6eba004d-cb3f-459b-9c39-1d5f656d61d7.png">.

Ends up looking like this: 

<img width="800" alt="image" src="https://user-images.githubusercontent.com/22749289/155608851-23f25ef7-2c65-4bcf-80fb-fc8f86fc61af.png">.

Is this possible in igv.js? I just found this today, so may be missing something straightforward. Thanks! :) 

<details>

<summary>For example, here's a full working example of a webpage with two tracks (Test_5 and Test_3) that I would like to combine:</summary>

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <link href=img/favicon.ico rel="shortcut icon">
    <title>igv.js</title>

    <!-- IGV JS -->
    <script src="https://cdn.jsdelivr.net/npm/igv@2.10.5/dist/igv.min.js"></script>
</head>

<body>

<h1>VCF variants and CRAM alignments</h1>

<div id="igvDiv" style="padding-top: 10px;padding-bottom: 10px; border:1px solid lightgray"></div>

<script type="text/javascript">

    document.addEventListener("DOMContentLoaded", function () {

        var options =
            {
                // Example of fully specifying a reference .  We could alternatively use  "genome: 'hg19'"
                reference:
                    {
                        id: "NC_000913_2",
                        fastaURL: "https://20220224coligenome.s3.amazonaws.com/nc_000913_2.fasta",
                        cytobandURL: "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/b37/b37_cytoband.txt"
                    },
                tracks:
                    [

                        {
                            name: "Genes",
                            type: "annotation",
                            format: "bed",
                            url: "https://20220224coligenome.s3.amazonaws.com/E_coli_NC_000913.2_all_20200422.bed",
                            order: Number.MAX_VALUE,
                            visibilityWindow: 3000000,
                            displayMode: "EXPANDED",
                            searchable: true
                        } ,
                        {
                            name: "Test_3",
                            type: "wig",
                            url: "https://20220224coligenome.s3.amazonaws.com/171030Li_Ecoli_WT_rend_3_f.bw",
                            displayMode: "EXPANDED",
                            searchable: true
                        },
                         {
                            name: "Test_5",
                            type: "wig",
                            url: "https://20220224coligenome.s3.amazonaws.com/171030Li_Ecoli_WT_rend_5_f.bw",
                            displayMode: "EXPANDED",
                            searchable: true
                        }
                    ]

            };

        var igvDiv = document.getElementById("igvDiv");

        igv.createBrowser(igvDiv, options)
            .then(function (browser) {
                console.log("Created IGV browser");
            })
    })
</script>

</body>

</html>
```

</details>

_Originally posted by @julianstanley in https://github.com/igvteam/igv.js/issues/1460#issuecomment-1050272631_

**Comment by julianstanley** [comment]
Woop, redundant with #1154, sorry about that. Closing.

**Comment by jrobinso** [comment]
We are currently working on exactly this feature.

**Comment by jrobinso** [comment]
@julianstanley You can do this from the track configuration,  I assume you already know that but here is an example:  https://github.com/igvteam/igv.js/blob/master/dev/misc/merged.html.   Sorry if I misinterpreted you initially, I thought you were requesting to do this through the track menu.   That is what we are working on.

**Comment by julianstanley** [comment]
> @julianstanley You can do this from the track configuration,  I assume you already know that but here is an example:  https://github.com/igvteam/igv.js/blob/master/dev/misc/merged.html.   Sorry if I misinterpreted you initially, I thought you were requesting to do this through the track menu.   That is what we are working on.

Thanks, Jim! 

When I first posted this issue I was looking for the track configuration, which is now working beautifully (when I closed the issue, I noticed that track config was already implemented and track menu was in progress). Hopefully your link above will help others looking for the same thing. 

Thanks again for your work improving and maintaining this, I think it's going to be really helpful for sharing information in my lab (since not everyone uses IGV, it'll be much easier to just send them a URL).
